tests/fast-integration/skr-nightly/test/test-nightly.js

```diff
@@ -0,0 +1,58 @@
+const {
+   KCPConfig,
+   KCPWrapper,
+} = require('../../kcp/client');
+
+const {
+   KEBConfig,
+   KEBClient,
+} = require('../../kyma-environment-broker');
+
+const {
+   GardenerClient,
+   GardenerConfig,
+} = require('../../gardener');
+
+const {initializeK8sClient} = require('../../utils');
+
+const {
+   commerceMockTests,
+   gettingStartedGuides,
+} = require('../../test');
+
+describe(`SKR Nightly periodic test`, function () {
+   const kebconfig = KEBConfig.fromEnv();
+   const gardenerConfig = GardenerConfig.fromEnv();
+
+   const gardener = new GardenerClient(gardenerConfig);
+   const keb = new KEBClient(kebconfig);
+
+   process.env.KCP_KEB_API_URL = `https://kyma-env-broker.` + keb.host;
+   process.env.KCP_GARDENER_NAMESPACE = `garden-kyma-dev`;
+   process.env.KCP_OIDC_ISSUER_URL = `https://kymatest.accounts400.ondemand.com`;
+
+   const config = KCPConfig.fromEnv();
+   const kcp = new KCPWrapper(config);
+
+   let runtime;
+
+   describe(`Prepare kube client`, function () {
+      it(`Fetch last runtime`, async function() {
+         await kcp.login()
+         let query = {
+            subaccount: keb.subaccountID,
+         }
+         let runtimes = await kcp.runtimes(query);
+         if (runtimes.data) {
+            runtime = runtimes.data[0];
+         }
+      });
+      it (`Initialize k8s client from nightly runtime`, async function () {
+         const shoot = await gardener.getShoot(runtime.shootName);
+         console.log(shoot.name, runtime.shootName);
+         initializeK8sClient({ kubeconfig: shoot.kubeconfig });
+      });
+   });
+   commerceMockTests();
```

Comment:
You need to use compass test fixture like here:
https://github.com/kyma-project/kyma/blob/cfa81506876c3037048dedbf0f01c557ba127888/tests/fast-integration/skr-test/skr-test.js#L160-L168

Reply:
So, basically the tests that are defined in `test/` directory are not meant for SKR?

Reply:
In this case maybe refactoring the `skr-test` to be more re-usable will be better, in similar I did with the regular tests? Because right now the test cases are copy-pasted to the nightly provision job and run just after the SKR is provisioned.